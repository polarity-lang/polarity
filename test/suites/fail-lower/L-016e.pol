data Bool { T, F }
let Type : Bool { T }