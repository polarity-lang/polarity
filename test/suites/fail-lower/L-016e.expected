L-016

  × "Type" is not a valid identifier
   ╭─[L-016e.pol:2:1]
 1 │ data Bool { T, F }
 2 │ let Type : Bool { T }
   · ─────────────────────
   ╰────
  help: "Type" is the name of the impredicative type universe.
